app/src/androidTest/java/de/test/antennapod/service/download/DownloadServiceTest.java

```diff
@@ -17,10 +17,8 @@
 import org.junit.runner.RunWith;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
-import java.util.concurrent.CountDownLatch;
```

Comment:
Werid, the checksum between this file and the one on branch `develop` is the same not sure why it's showing a diff

Reply:
Hmm, maybe squash the commits together and rebase?

Reply:
sorry, my branch is a message, can you review https://github.com/AntennaPod/AntennaPod/pull/4895/files